### PR TITLE
Add option to set recursion limit

### PR DESCRIPTION
**Pull request by wing328**
- Add an option to `setrecursionlimit` in `__init__.py`
- tested locally in API and model tests to confirm the recursion limit is set correctly to the new value provided in the option

### PR checklist
 
- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] Pull Request title clearly describes the work in the pull request and Pull Request description provides details about how to validate the work. Missing information here may result in delayed response from the community.
- [x] If contributing template-only or documentation-only changes which will change sample output, [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) beforehand.
- [x] Run the shell script `./bin/generate-samples.sh`to update all Petstore samples related to your fix. This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit as it would merge with master. These must match the expectations made by your contribution. You may regenerate an individual generator by passing the relevant config(s) as an argument to the script, for example `./bin/generate-samples.sh bin/configs/java*`. For Windows users, please run the script in [Git BASH](https://gitforwindows.org/).
- [x] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`
- [x] Copy the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

cc @taxpon (2017/07) @frol (2017/07) @mbohlool (2017/07) @cbornet (2017/09) @kenjones-cisco (2017/11) @tomplus (2018/10) @Jyhess (2019/01) @arun-nalla (2019/11) @spacether (2019/11)

### CODE REVIEW

File: docs/generators/python-experimental.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ sidebar_label: python-experimental
 |packageUrl|python package URL.| |null|
 |packageVersion|python package version.| |1.0.0|
 |projectName|python project name in setup.py (e.g. petstore-api).| |null|
+|setRecursionLimit|Set recursion limit. If not set, use the system default value.| |null|
```
Comment (spacether) [inline docs/generators/python-experimental.md:15]: What do you think about naming it recursionLimit?
other parameters only have verb prefixes if they are booleans. If you want to keep it as is that's fine also.

Comment (wing328) [reply docs/generators/python-experimental.md:15]: Thanks for the feedback. Let me revise it.

Comment (wing328) [reply docs/generators/python-experimental.md]: Updated via https://github.com/OpenAPITools/openapi-generator/pull/7491/commits/d84cdc50619e7284eb7c9ddc4d28ac0e3b143f06

**Review by spacether — APPROVED** [review]
Thank you for the name change; looks good to me!

**Pull request merged**